src/main/java/net/earthcomputer/clientcommands/command/AliasCommand.java

```
@@ -0,0 +1,174 @@
+            Gson gson = new Gson();
+
+            gson.toJson(aliasMap, writer);
+            writer.flush();
+            writer.close();
+
+            sendFeedback(new LiteralText("New alias added successfully!"));
+        } catch (Exception e) {
+            sendError(new LiteralText("Uh oh"));
+        }
+        return 0;
+    }
+    private static int listAliases() {
+        HashMap<String, String> aliasMap = getAliases();
+        if (aliasMap == null || aliasMap.isEmpty()) {
+            sendError(new LiteralText("No aliases registered"));
+        } else {
+            sendFeedback(new LiteralText("Aliases registered: " + aliasMap.size()));
+
+            for(String key: aliasMap.keySet()) {
+                sendFeedback(Formatting.BOLD + key + Formatting.RESET+ ": "+ aliasMap.get(key).replace("%","%%"));
+            }
+
+        }
+        return 0;
+    }
+    private static int removeAlias(String key) {
+        ClientPlayNetworkHandler networkHandler = MinecraftClient.getInstance().getNetworkHandler();
+        assert networkHandler != null;
+        var dispatcher = (CommandDispatcher<ServerCommandSource>) (CommandDispatcher<?>) networkHandler.getCommandDispatcher();
+
+        HashMap<String, String> aliasMap = getAliases();
+
+        if(aliasMap != null && aliasMap.containsKey(key)) {
+            BrigadierRemover.of(dispatcher).get(key).remove();
+            aliasMap.remove(key);
+        } else {
+            sendError(new LiteralText("No such key exists"));
+            return 0;
+        }
+
+        try {
+            Writer writer = new FileWriter(String.valueOf(aliasPath));
+            Gson gson = new Gson();
+
+            gson.toJson(aliasMap, writer);
+            writer.flush();
+            writer.close();
+            sendFeedback(new LiteralText("Alias removed successfully!"));
+        } catch (Exception e) {
+            sendError(new LiteralText("Uh oh"));
```

You should probably throw an exception rather than sending an error message.